modules/visual_script/visual_script_editor.cpp

```diff
@@ -3009,7 +2962,7 @@ void VisualScriptEditor::_graph_connect_to_empty(const String &p_from, int p_fro
 	if (!vsn.is_valid()) {
 		return;
 	}
-	if (vsn->get_output_value_port_count()) {
+	if (vsn->get_output_value_port_count() || vsn->get_output_sequence_port_count()) {
```

Comment:
That's this additional change, please test if it fixes what @DavidCambre reported.

Reply:
Tested. OK